src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -87,6 +87,8 @@ internal class WorkflowInstance : TaskScheduler, IWorkflowInstance, IWorkflowCon
         private WorkflowUpdateDefinition? dynamicUpdate;
         private bool workflowInitialized;
         private bool applyModernEventLoopLogic;
+        private WorkflowDefinitionOptions definitionOptions;
```

Comment:
```suggestion
        private readonly WorkflowDefinitionOptions definitionOptions;
```

(and move to the readonly set of fields, unless I'm missing something)